phpstan.neon

```diff
@@ -102,6 +102,9 @@ parameters:
         - message: '#Cannot call method addFunction\(\) on EonX\\EasyDecision\\Expressions\\Interfaces\\ExpressionLanguageInterface\|null#'
           path: packages/EasyDecision/tests/Bridge/Symfony/Stubs/Configurators/TypeRestrictedExpressionFunctionConfigurator.php
 
+        - message: '#Parameter \#1 \$rules of class EonX\\EasyDecision\\Configurators\\AddRulesDecisionConfigurator constructor expects iterable<EonX\\EasyDecision\\Interfaces\\RuleInterface\>, array<int, class@anonymous/EasyDecision/tests/Configurators/AddRulesDecisionConfiguratorTest\.php\:75\|EonX\\EasyDecision\\Tests\\Stubs\\RestrictedRuleStub\> given#'
```

Comment:
Instead of adding this, you should modify the phpdoc of the configurator to accept mixed, because it's the truth, it can deal with anything thanks to `filterRules()`